commands/new.go

```diff
@@ -0,0 +1,169 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/go-morph/morph"
+	"github.com/spf13/cobra"
+
+	. "github.com/dave/jennifer/jen"
+)
+
+const baseDriverPath = "drivers"
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "new",
+		Short: "Bootstraps new resources",
+	}
+
+	cmd.AddCommand(
+		NewDriverCmd(),
+	)
+
+	return cmd
+}
+
+func NewDriverCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:           "driver",
+		Short:         "Generates necessary file structure for a new driver",
+		Args:          cobra.ExactArgs(1),
+		RunE:          newDriverCmdF,
+		SilenceUsage:  true,
+		SilenceErrors: true,
+	}
+}
+
+func newDriverCmdF(cmd *cobra.Command, args []string) error {
+	driverName := args[0]
+	morph.InfoLogger.Printf("Generating necessary file structure for %q driver\n", driverName)
+
+	if _, err := os.Stat(baseDriverPath + "/" + driverName); !os.IsNotExist(err) {
+		return fmt.Errorf("driver %q already exists, skipping", driverName)
+	}
+
+	return generateNewDriver(driverName)
+}
+
+func generateNewDriver(driverName string) error {
+	morph.InfoLoggerLight.Printf("== CreateDriver: %q: generating driver files\n", driverName)
+	morph.InfoLoggerLight.Println("=================================================")
+
+	f := NewFile(driverName)
+
+	f.PackageComment("Initial code generated by generator.")
+
+	f.ImportNames(map[string]string{
+		"github.com/go-morph/morph/drivers": "drivers",
+		"github.com/go-morph/morph/models":  "models",
+	})
+
+	f.Var().Add(Id("driverName")).Op("=").Add(Lit(driverName))
+	f.Var().Add(Id("defaultConfig")).Op("=").Add(Op("&")).Id("Config").Values(Dict{
+		Id("MigrationsTable"): Lit("schema_migrations"),
+	})
+	f.Comment("add here any custom driver configuration")
+	f.Var().Add(Id("configParams")).Op("=").Add(Index().String().Values())
+
```

Comment:
I don't know how precise/verbose we'd like to be with the `f.Line()` calls, but I'd add one after the imports and another one here to sepparate vars from the init func

Reply:
Imports are automatically handled, so the top of the file looks good i think:

```go
// Initial code generated by generator.
package mysql

import (
	"github.com/go-morph/morph/drivers"
	"github.com/go-morph/morph/models"
)

var driverName = "mysql"
var defaultConfig = &Config{MigrationsTable: "schema_migrations"}

// add here any custom driver configuration
var configParams = []string{}

func init() {
	drivers.Register("mysql", &mysql{})
}

type Config struct {
	MigrationsTable string
	// Add more properties here
}

type mysql struct {
	config *Config
	// Add more properties here
}

func WithInstance(dbInstance interface{}, config *Config) (drivers.Driver, error) {
	return &mysql{config: config}, nil
}
```

Reply:
:heart_eyes: